### PR TITLE
IBX-822: Docbook format

### PR DESCRIPTION
**Pull request by DominikaK**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | https://issues.ibexa.co/browse/IBX-822
| Versions      | 2.5+

### CODE REVIEW

File: docs/api/field_types_reference/richtextfield.md [hunk 1]
```diff
@@ -8,29 +8,38 @@ This Field Type validates and stores structured rich text, and exposes it in sev
...
-Currently supported input formats are described in the table below:
+!!! note "Legacy eZXML format"
 
-|Name|Description|
-|------|------|
-|[[= product_name =]]'s DocBook variant|Field Type's internal format.|
-|XHTML5 editing format|Typically used with in-browser HTML editor.|
-|Legacy eZXML format|Compatibility with legacy XML format, used by [XmlText Field Type](xmltextfield.md).|
+    The Legacy eZXML format is also supported by the RichText Field Type,
+    but can be used only for migration, in compatibility with the [legacy XML format](xmltextfield.md).
```
Comment (konradoboza) [inline docs/api/field_types_reference/richtextfield.md:38]: Maybe we can emphasize what tools should be used to perform the migration, ref: https://github.com/ezsystems/ezplatform-xmltext-fieldtype#migrating-from-xmltext-to-richtext (if this has not been mentioned already)?

File: docs/api/public_php_api_creating_content.md [hunk 2]
```diff
@@ -82,15 +82,16 @@ based on the input file.
...
-You can see more information about this format in [Field Types reference](field_types_reference/richtextfield.md#example-of-the-field-types-internal-format).
-
+The RichText Field accepts values in a custom flavor of [Docbook](https://github.com/docbook/wiki/wiki) format.
 For example, to add a simple RichText paragraph, provide the following as input:
 
 ``` xml
 <section xmlns="http://docbook.org/ns/docbook" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:ezxhtml="http://ez.no/xmlns/ezpublish/docbook/xhtml" xmlns:ezcustom="http://ez.no/xmlns/ezpublish/docbook/custom" version="5.0-variant ezpublish-1.0"><para>Description of your Content item.</para></section>
 ```
 
+To learn more about the format and how ir represents different elements of rich text, see
```
Comment (konradoboza) [inline docs/api/public_php_api_creating_content.md:92]:
```suggestion
To learn more about the format and how it represents different elements of the RichText, see
```

**Review by konradoboza — APPROVED** [review]

File: docs/api/field_types_reference/richtextfield.md (hunk 1 above)
Comment (DominikaK) [reply docs/api/field_types_reference/richtextfield.md:38]: @konradoboza I guess we could link to https://doc.ibexa.co/en/master/migrating/migrating_from_ez_publish_platform/#321-migrate-xmltext-content-to-richtext instead?

Comment (konradoboza) [reply docs/api/field_types_reference/richtextfield.md:38]: Sure, let's go with that reference.

File: docs/api/public_php_api_creating_content.md [hunk 3]
```diff
@@ -82,15 +82,16 @@ based on the input file.
...
-
+The RichText Field accepts values in a custom flavor of [Docbook](https://github.com/docbook/wiki/wiki) format.
 For example, to add a simple RichText paragraph, provide the following as input:
 
 ``` xml
 <section xmlns="http://docbook.org/ns/docbook" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:ezxhtml="http://ez.no/xmlns/ezpublish/docbook/xhtml" xmlns:ezcustom="http://ez.no/xmlns/ezpublish/docbook/custom" version="5.0-variant ezpublish-1.0"><para>Description of your Content item.</para></section>
 ```
 
+To learn more about the format and how it represents different elements of rich text, see
+[RichText Field Type reference](field_types_reference/richtextfield.md#custom-docbook-format).
```
Comment (juskora) [inline docs/api/public_php_api_creating_content.md:93]: Is this link correct? should it redirect to Docbook format or to RichText Reference page? Now it leads to Docbook format.

**Review by juskora — CHANGES_REQUESTED** [review]

File: docs/api/public_php_api_creating_content.md (hunk 3 above)
Comment (DominikaK) [reply docs/api/public_php_api_creating_content.md:93]: Yes, that is the intention, to point directly to the ref. for specific docbook elements

**Review by juskora — APPROVED** [review]

**Pull request merged**